joomla/joomla-cms
# [5.0] Finally delete deprecated javascript assets from PR's #38823 and #39374

Pull Request for [Issue #](https://github.com/joomla/joomla-cms/pull/40296#issuecomment-1493329420) .

### Summary of Changes

This pull request (PR) finally deletes the js files and the asset which had once been deleted with PR's #38823 and #39374 and then added back with PR #39431 for b/c with old layout overrides.

It is a simple revert of PR #39431 in the 5.0-dev branch plus adding the deleted compiled and minified js files from the media folder on update, i.e. adding them to the deleted files list in "script.php".

The deprecated language strings used by the removed js will be removed with PR #40296 .

### Testing Instructions

Code review: Verify that this PR here is exactly reverting PR #40296 plus adding the deleted compiled and minified js files to the deleted files list in "script.php".

In addition, test an update from the latest 4.4-dev nightly build to the update package or custom URL created by Drone for this PR.

### Actual result BEFORE applying this Pull Request

Deprecated js files and asset from PR #40296 are still present in the 5.0-dev branch. 

### Expected result AFTER applying this Pull Request

Deprecated js files and asset from PR #40296 are removed from the 5.0-dev branch, and the compiled and minified js files are deleted on update from 4.4-dev. 

### Link to documentations
Please select:

- [X] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/98
